src/settings/JobProfiles/ViewJobProfile/ViewJobProfile.js

```diff
@@ -169,6 +170,8 @@ const ViewJobProfileComponent = props => {
   };
 
   const handleRun = async record => {
+    setIsConfirmButtonDisabled(true);
```

Comment:
Should the `isConfirmButtonDisabled` state be reset somewhere?

Reply:
@Dmytro-Melnyshyn Good catch! Thank you! Added reset the state for `isConfirmButtonDisabled` when we catch the error while trying to run import